anomaly_detector/anomaly_detector.py

```diff
@@ -123,11 +126,11 @@ def train(self):
         dist = []
         cnt = 0
         total = len(to_put_train)
-        for i in to_put_train:
-            if not cnt % int(total/10):
-                _LOGGER.info("Anomaly scoring %d/%d" % (cnt, total))
-            dist.append(self.model.get_anomaly_score(i))
-            cnt += 1
+
+        pool = Pool()
```

Comment:
Can you extract this block and the other `Pool` use into a function which would accept a list as a parameter so that we don't have 2 copies of the same code here?

Also, we should probably introduce a parameter which would set number of processes we want to use, because `Pool` uses `os.cpu_count()` to get this number and Python does not respect OpenShift resource requests/limits and this is what you get in a container:

```
>>> import os
>>> os.cpu_count()
56
```

Which would not be very effective:) I talk about CPU limits elsewhere in comments as well, so probably a single parameter and env var would do for `Pool(processes=)` and `n_job` in Sompy